examples/get-started/first_figure.py

```diff
@@ -91,8 +91,9 @@
 # Set the projection
 # ------------------
 #
-# This figure now has its colors set, but there is no projection or size
-# set for the map. Both of these values are set using the ``projection``
+# This figure now has its colors set. By default the projection and size
+# of the map is set to ``"JQ15c+``". However, both of these values can be
```

Comment:
```suggestion
# of the map is set to ``"Q15c+du``". However, both of these values can be
```

Reply:
Better to explain the meaning of `Q15c+du`.

Reply:
Is it really ``+du`` by default @seisman ? Based on https://docs.generic-mapping-tools.org/dev/cookbook/map-projections.html it should be ``+dw`` shouldn't it?